frontend/src/metabase/query_builder/components/expressions/ExpressionEditorSuggestions.jsx

```diff
@@ -79,6 +83,10 @@ export default class ExpressionEditorSuggestions extends Component {
     event.preventDefault();
     event.stopPropagation();
 
+    if (event.target.tagName === "A" || event.target.tagName === "BUTTON") {
+      return;
+    }
```

Comment:
Interacting with anything in the popover should not cause the related suggestion to get selected.